graphz/src/test/scala/coop/rchain/graphz/GraphzSpec.scala

```diff
@@ -1,13 +1,14 @@
 package coop.rchain.graphz
 
-import cats._, cats.data._, cats.implicits._
-import cats.mtl._
+import cats._
+import cats.data._
 import cats.mtl.implicits._
+
 import org.scalatest._
 
 class GraphzSpec extends FunSpec with Matchers with BeforeAndAfterEach with AppendedClues {
 
-  type Effect[A] = StateT[Id, StringBuffer, A]
+  type Effect[A] = StateT[Eval, StringBuffer, A]
```

Comment:
Why not Coeval? Monix docs have:

```
at the moment of writing this, the Monix Coeval takes care of error handling, while the Cats Eval does not, providing operations for recovery, thus also working well as a replacement for Scala’s Try type.
```

How does Eval handle errors in the wrapped code? Will `.value` simply throw an exception?

Reply:
@ArturGajowy I see your point. But let's stick to `cats` for now. All of our current `GraphSerializer` instances are pure and won't throw any (non-fatal) exceptions.

Reply:
Irrelevant side note: `StateT[Eval, A, B]` is just `State[A, B]`

Reply:
@marcin-rzeznicki are you sure that State[A, B] is stacksafe?

Reply:
Yeah: 
https://github.com/typelevel/cats/blob/8c031e5a4ed63a8efc34c196f95ccfcfea7eeb77/core/src/main/scala/cats/data/package.scala#L58

https://github.com/typelevel/cats/blob/ed99e170ee65dfe9d03995dd72b33ec90ae9f926/tests/src/test/scala/cats/tests/IndexedStateTSuite.scala#L40

Reply:
Had it used `Id` then obviously it wouldn't have been stack safe. It strongly relies on stack-safety of the underlying `F` ie.
https://github.com/typelevel/cats/blob/ed99e170ee65dfe9d03995dd72b33ec90ae9f926/core/src/main/scala/cats/data/IndexedStateT.scala#L25

Reply:
Oh, cool! Wasn't aware of the Eval in there :)